pkg/operators/cluster-autoscaler-operator.go

```
@@ -110,3 +117,322 @@ var _ = Describe("Cluster autoscaler cluster operator status should", framework.
+					"ECDHE-RSA-AES128-GCM-SHA256",
+				},
+			},
+		},
+	}
+}
+
+func getAPIServer(ctx context.Context, client runtimeclient.Client) *configv1.APIServer {
+	apiServer := &configv1.APIServer{}
+	key := types.NamespacedName{Name: apiServerName}
+	ExpectWithOffset(1, client.Get(ctx, key, apiServer)).To(Succeed(), "Failed to get APIServer")
+	return apiServer
```

golangci-lint (2.12.2)

[error] return with no blank line before

(nlreturn)